ReSharper.FSharp/src/FSharp.Psi.Intentions/src/QuickFixes/UnionCaseDoesNotTakeArgumentsFix.fs

```diff
@@ -0,0 +1,36 @@
+namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Highlightings
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Impl
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Impl.Tree
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Util
+open JetBrains.ReSharper.Psi.ExtensionsAPI
+open JetBrains.ReSharper.Psi.ExtensionsAPI.Tree
+open JetBrains.ReSharper.Resources.Shell
+
+type UnionCaseDoesNotTakeArgumentsFix(error: UnionCaseDoesNotTakeArgumentsError) =
+    inherit FSharpQuickFixBase()
+
+    let pat = error.Pattern.As<IParametersOwnerPat>()
+
+    override x.Text = "This union case does not take arguments"
```

Comment:
This text looks like a warning text, not like a quick fix one. Let's use a descriptive text here, please?

Reply:
I think this quick fix should also work with other symbols, e.g. enum values or literals:

```fsharp
type E =
    | A = 1
    | B = 2

match E.A with
| E.A x -> ()
```
<img width="371" alt="Screenshot 2022-11-22 at 18 48 25" src="https://user-images.githubusercontent.com/3923587/203385464-77946543-af75-41a9-a05c-7b304a253b22.png">

```fsharp
let [<Literal>] A = 1

match 1 with
| A x -> ()
```

<img width="363" alt="Screenshot 2022-11-22 at 18 48 18" src="https://user-images.githubusercontent.com/3923587/203385471-589d1b38-547b-4dd4-83e8-19c0fed66f59.png">

So the name shouldn't mentions unions cases in particular. Would be nice to have tests for these too. 🙂 

Maybe something like `"Remove argument"` and  `RemovePatternArgumensFix`?

Reply:
Ohh nice :). I did not realize that. I will add unit test. Thanks for the suggestion

Reply:
Updated and Added unit tests